### PR TITLE
Yy createbrowsescreen

### PR DESCRIPTION
**Pull request by yenyin262**
- Implement images and navigate functionality

### CODE REVIEW

File: client/js/screens/Browse/Browse.js [hunk 1]
```diff
@@ -1,10 +1,61 @@
...
+              source={require('../../assets/shoes.png')}
+              style={styles.browseImage}
+            />
+            <Text style={styles.text}> Shoes </Text>
+          </TouchableOpacity>
+        </View>
+        <View>
+          <TouchableOpacity
+            onPress={() => {
+              // navigation.navigate('items', {catgeory: 'bags'});
```
Comment (iyuvraajsingh) [inline client/js/screens/Browse/Browse.js]: Please remove unwanted code.

**Review by iyuvraajsingh — COMMENTED** [review]
Please remove unwanted code.

File: client/js/screens/Browse/Browse.js [hunk 2]
```diff
@@ -1,10 +1,61 @@
 import React from 'react';
-import {View, Text} from 'react-native';
+import {View, Text, Image, ScrollView, TouchableOpacity} from 'react-native';
+import styles from './styles';
 
-const Browse = () => {
+const Browse = ({navigation}) => {
   return (
     <View>
```
Comment (siddharthparmar7) [inline client/js/screens/Browse/Browse.js:7]: No need to wrap your scroll view with the view component. Please remove View component.

**Review by iyuvraajsingh — APPROVED** [review]

File: client/js/screens/Browse/styles.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
...
+    fontSize: 18,
+    marginHorizontal: 10,
+  },
+  browseImage: {
+    height: 180,
+    width: '94%',
+    alignItems: 'center',
+    justifyContent: 'center',
+    marginHorizontal: 10,
+    marginTop: 10,
```
Comment (siddharthparmar7) [inline client/js/screens/Browse/styles.js:14]: use flexbox properties, resizeMode, use %s for height, width, margin and padding.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]

**Pull request merged**